docs/spec/bank/keepers.md

```diff
@@ -0,0 +1,130 @@
+## Keepers
+
+The bank module provides three different exported keeper interfaces which can be passed to other modules which need to read or update account balances. Modules should use the least-permissive interface which provides the functionality they require.
```

Comment:
I feel like a warning is due somewhere that sendkeeper is being given permissions to mint coins, and you have to trust that its not / review the code yourself. (Or alternatively we produce a diagram of permissions to mint coins)

Reply:
Hmm I guess, but this will be true of *every* least-permissioned keeper we write (at least, every one which does anything interesting) - they'll always have more-permissioned unexposed sub-keepers.

Reply:
Added a warning to always review the code.